app/controllers/organisations_controller.rb

```diff
@@ -0,0 +1,23 @@
+class OrganisationsController < ApplicationController
+  def index; end
+
+  def edit
+    @organisation = Organisation.find_by(id: params.fetch(:id))
+  end
+
+  def update
+    organisation = Organisation.find_by(id: params.fetch(:id))
+    if organisation.update_attributes(organisartion_params)
+      flash[:notice] = 'Organisation updated'
+      redirect_to organisations_path
+    else
+      flash[:alert] = 'Update failed'
```

Comment:
Doesn't handle the failure properly, when I save the form with an empty service email it gives me a green success banner and takes me back to the show page. It should re-render the edit page and show the errors in the form.